bibliography.bib

```diff
@@ -546,6 +546,24 @@ @Article{clusterdata:Mishra2010
 
 ################ 2020
 
+@INPROCEEDINGS{clusterdata:Lin2020,
+   title = {Using GANs for Sharing Networked Time Series Data: Challenges, Initial Promise, and Open Questions},
```

Comment:
s/GANs/{GANs}/ to keep the capitalization correct.

Reply:
Done.